### ISSUE TITLE
JavaScript vs ECMAScript

### ISSUE THREAD

**Issue by lukescott** [lukescott/es-type-hinting#28]
This is a tough one. The language is commonly known as JavaScript, but it's more of a nickname, right? The proper name is ECMAScript, so we should probably stick with that in the document.

**Comment by Naddiseo** [comment]
Indeed. I think because this is a spec for "es"-type-hinting, we should go with ECMAScript. Perhaps see what other proposals use.
